Tweak.xm

```diff
@@ -12,6 +12,7 @@
 - (instancetype)initWithContentSource:(AVPictureInPictureControllerContentSource *)contentSource;
 - (void)sampleBufferDisplayLayerRenderSizeDidChangeToSize:(CGSize)renderSize;
 - (void)sampleBufferDisplayLayerDidAppear;
+- (void)_stopPictureInPictureAndRestoreUserInterface:(BOOL)arg1;
```

Comment:
Is this used?

Reply:
> Is this used?

It was used when I was testing a bunch of stuff, but it can be removed.